src/editor/codemirror/themeExtensions.ts

```diff
@@ -82,13 +82,13 @@ export const themeExtensions = (fontSize: string) => {
     },
     ".cm-tooltip.cm-completionInfo": {
       width: "20rem",
-      height: "10rem",
+      height: "10.5rem",
```

Comment:
Not confident in this hight. Might also look at the font size choices for the docs.

Reply:
I think this is OK having lived with it for a while.